### DISCUSSION TITLE
generating csv file (ray tracing)

### DISCUSSION THREAD

**Discussion by hadiaushaq** [NVlabs/sionna#635] [Q&A] [answered]
I have generated a ray-traced coverage map for a specific location using ray tracing model. How can I export the longitude and latitude values along with the corresponding RSSI values for each pixel in the map to a CSV file

<img width="904" alt="Screenshot 2024-10-29 135943" src="https://github.com/user-attachments/assets/8f30625c-0c05-456c-a6b9-1d7083fbc2f2">

**Comment by faycalaa** [comment, marked as answer]
Hi @hadiaushaq,

Sionna doesn't have a built-in feature for exporting coverage maps as CSV files.

However, you can access the computed RSS values using the [`rss` property of the `CoverageMap` object](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.CoverageMap.rss) and the [positions of the cell centers](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.CoverageMap.cell_centers). You could then save these values as a CSV file, e.g., using [Python's csv module](https://docs.python.org/3/library/csv.html).

**Reply by hadiaushaq** [reply]
Thank you so much for explaining it, I will go through it and let you know if I have more questions.

**Reply by merlinND** [reply]
Closing for now, @hadiaushaq feel free to reopen if you have another related question.

**Reply by hadiaushaq** [reply]
I am not able to use sample_position command in my code

**Reply by merlinND** [reply]
Hello @hadiaushaq,

Please open a new issue with a complete description of your problem, including a minimal but complete script that will allow us to reproduce it.
